test/jdk/javax/swing/JToolBar/bug4138694.java

```diff
@@ -39,19 +38,20 @@
 import javax.swing.SwingUtilities;
 
 public class bug4138694 {
+    public static final String exampleText = "Action";
```

Comment:
```suggestion
    private static final String actionName = "Action";
```
It fits better this way, but it's unimportant.

Reply:
Updated. Thanks